lib/bundler/remote_specification.rb

```diff
@@ -75,11 +75,7 @@ def _remote_specification
     end
 
     def method_missing(method, *args, &blk)
-      if Gem::Specification.new.respond_to?(method)
-        _remote_specification.send(method, *args, &blk)
-      else
-        super
-      end
+      _remote_specification.send(method, *args, &blk)
```

Comment:
https://github.com/bundler/bundler/commit/c563003292bd36dc647b41016fad93d5b7324a07 introduced it. We can also just use `Gem::Specification.instance_methods.any? {|m| m.to_sym == method.to_sym }` or something if it's actually important